pkg/apiserver/webhooks/datavolume-validate.go

```diff
@@ -54,6 +54,19 @@ func validateSourceURL(sourceURL string) string {
 	return ""
 }
 
+func (wh *dataVolumeValidatingWebhook) validateDataVolumeName(request *v1beta1.AdmissionRequest, name string) []metav1.StatusCause {
```

Comment:
There is no reason to make this function part of the struct, it doesn't use the struct in any way. validateDataVolumeSpec does use the wh struct, which is why it is part of it.

Reply:
you are right. Fixing that.